### PR TITLE
V233 preparation

### PR DESCRIPTION
**Pull request by poettering**
if you have fixes for NEWS, please file a follow-up PR, after all a typo isn't really a bug, and I'd rather have to go this in quickly. Thank you for understanding!

### CODE REVIEW

File: NEWS [hunk 1]
```diff
@@ -26,19 +28,330 @@ CHANGES WITH 233 in spe
...
+
         * Services of Type=notify require a READY=1 notification to be sent
           during startup. If no such message is sent, the service now fails,
           even if the main process exited with a successful exit code.
 
-        * The option MulticastDNS= of network configuration files has got
-          actual implementation. With MulticastDNS=yes a host can resolve
-          names of remote hosts and to reply to mDNS's A and AAAA requests
-          from the hosts.
+        * The option MulticastDNS= of network configuration files has acquire
```
Comment (martinpitt) [inline NEWS:38]: acquire"d"

File: NEWS [hunk 2]
```diff
@@ -26,19 +28,330 @@ CHANGES WITH 233 in spe
...
+          and actual implementation. With MulticastDNS=yes a host can resolve
```
Comment (martinpitt) [inline NEWS:39]: s/and/an/

Comment (zonque) [inline NEWS:39]: "acquire and" → "acquired an"

File: NEWS [hunk 3]
```diff
@@ -26,19 +28,330 @@ CHANGES WITH 233 in spe
...
+
+          x-systemd.after= and x-systemd.before= may be use to explicitly order
+          a mount after or before another unit or mount point.
+
+        * Enqueued start jobs for device units are now automatically garbage
+          collected if there are no jobs waiting for them anymore.
+
+        * systemctl list-jobs gained two new switches: --after and
+          --before. When specified for all queued jobs it is shown which other
+          queued jobs are waiting for it, or the job is waiting for.
```
Comment (martinpitt) [inline NEWS:93]: This is a bit hard to parse, I'll think about this.

File: NEWS [hunk 4]
```diff
@@ -26,19 +28,330 @@ CHANGES WITH 233 in spe
...
+
+        * systemd-nspawn gained support for ephemeral boots from disk images
+          (or in other words: --ephemeral and --image= may now be
+          combined). Moreover, ephemeral boots are now supported for normal
+          directories, even if the backing file system is not btrfs. Of course,
+          if the used file system does not support file system snapshots or
+          reflinks the initial copy operation will be relatively expensive, but
```
Comment (martinpitt) [inline NEWS:100]: comma after reflinks, otherwise this is hard to read

File: NEWS [hunk 5]
```diff
@@ -26,19 +28,330 @@ CHANGES WITH 233 in spe
...
+          this should still be suitable for many usecases.
```
Comment (martinpitt) [inline NEWS:101]: "use cases"

File: NEWS [hunk 6]
```diff
@@ -26,19 +28,330 @@ CHANGES WITH 233 in spe
...
+          repeated events has been added using the "/" character. For example,
+          "9..17/2:00" means "every two hours from 9am to 5pm".
+
+        * systemd-socket-proxyd gained a new parameter --connections-max= for
+          configuring the maximum number of concurrent connections.
+
+        * All python scripts shipped with systemd (specifically: the various
+          tests written in Python) now require Python 3.
+
+        * sd-id128 gained a new API for generating unique IDs for the host,
```
Comment (martinpitt) [inline NEWS:116]: drop the comma

File: NEWS [hunk 7]
```diff
@@ -26,19 +28,330 @@ CHANGES WITH 233 in spe
...
+          that do not leak the machine ID. Specifically,
```
Comment (martinpitt) [inline NEWS:117]: does not

File: NEWS [hunk 8]
```diff
@@ -26,19 +28,330 @@ CHANGES WITH 233 in spe
...
+          sd_id128_get_machine_app_specific() derives an ID based on the
+          machine ID in well-defined, non-reversible, stable way. This is
```
Comment (martinpitt) [inline NEWS:119]: in "a" well..

File: NEWS [hunk 9]
```diff
@@ -26,19 +28,330 @@ CHANGES WITH 233 in spe
...
+          with a temporary directory in the host's /var/tmp. This way changes
+          to /var are automatically flushed when the container shuts down.
+
+        * .network files gained a new Unmanaged= boolean setting for explicitly
+          excluding one or more interfaces from management by systemd-networkd.
+
+        * systemd-nspawn's disk image dissection code has been updated. Among
+          other things it's not permitted to pass raw file system block devices
+          to the --image= option (in addition to images containing partition
+          tables, as before).
```
Comment (martinpitt) [inline NEWS:152]: This should be written more concisely.

File: NEWS [hunk 10]
```diff
@@ -26,19 +28,330 @@ CHANGES WITH 233 in spe
...
+
+        * A new tool /usr/lib/systemd/systemd-dissect has been added that may
+          be used to dissect disk images the same way as systemd-nspawn does
+          it, following the Bootable Partition Specification. It may even be
+          used to mount disk images with complex partition setups (including
+          LUKS and Verity partitions) to a local host directory, in order to
+          inspect them. This tool is not considered public API (yet), and is
+          thus not installed into /usr/bin. Please do not rely on its
+          existance, since it might go away or be changed in later systemd
+          versions.
```
Comment (martinpitt) [inline NEWS:170]: I wonder if we should publicly advertise it already?

File: NEWS [hunk 11]
```diff
@@ -26,19 +28,330 @@ CHANGES WITH 233 in spe
...
+
+        * A new generator "systemd-verity-generator" has been added, similar in
+          style to "systemd-cryptsetup-generator" permitting automatic setup of
```
Comment (martinpitt) [inline NEWS:173]: comma before "permitting" for readability

File: NEWS [hunk 12]
```diff
@@ -26,19 +28,330 @@ CHANGES WITH 233 in spe
...
+          a disk image instead of plain directory. This logic reuses the same
+          image dissection and mount logic that systemd-nspawn already uses,
+          and hence supports any disk images systemd-nspawn supports, including
+          those following the Discoverable Partition Specification, as well as
+          Verity enabled images. This option enables systemd to run system
+          services directly off disk images acting as resource bundles,
+          possibly even including full integrity data.
+
+        * A new MountAPIVFS= unit file option has been added, taking a boolean
+          argument. If enabled /proc, /sys and /proc (collectively called the
```
Comment (martinpitt) [inline NEWS:289]: The second /proc should be /dev, according to the manpage.

File: NEWS [hunk 13]
```diff
@@ -26,19 +28,330 @@ CHANGES WITH 233 in spe
...
+          of coredumps in reverse order.
+
+        * The systemd-coredump logic has been improved so that it may be reused
+          for collecting backtraces in non-compiled languages, for example in
+          scripting languages such as Python.
+
+        * machinectl will now show the UID shift of local containers, if user
+          namespacing is enabled for them.
+
+        * systemd will not optionally run "environment generator" binaries at
```
Comment (martinpitt) [inline NEWS:318]: s/not/now/

File: NEWS [hunk 14]
```diff
@@ -26,19 +28,330 @@ CHANGES WITH 233 in spe
...
+          configuration load time. They may be used to add environment
+          variables to the environment block passed to services invoked. One
+          user environment generator is shipped by default, that sets up
```
Comment (martinpitt) [inline NEWS:321]: drop comma

File: NEWS [hunk 15]
```diff
@@ -26,19 +28,330 @@ CHANGES WITH 233 in spe
...
+        Rike-Benjamin Schuppner, Robert Kreuzer, Ronny Chevalier, Ruslan
+        Bilovol, sammynx, Sergey Ptashnick, Sergiusz Urbaniak, Stefan Berger,
+        Stefan Hajnoczi, Stefan Schweter, Susant Sahani, Sylvain Plantefève,
+        Taylor Smock, Thomas Blume, Thomas H. P. Andersen, Tobias Stoeckmann,
+        Tom Gundersen, Torstein Husebø, Viktar Vaŭčkievič, Viktor Mihajlovski,
+        Waldemar Brodkorb, Walter Garcia-Fontes, Wim de With, Yassine
+        Imounachen, Yi EungJun, Yu Watanabe, Zbigniew Jędrzejewski-Szmek,
+        Александр Тихонов
+
+        — Santa Fe, 2017-02-XX
```
Comment (martinpitt) [inline NEWS:353]: Berlin? :-)

**Comment by martinpitt** [comment]
Filed PR #5407 with NEWS fixes
